packages/smooth_app/lib/generic_lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -103,10 +103,16 @@ class SmoothAlertDialog extends StatelessWidget {
   }
 
   Padding _buildBottomBar(EdgeInsetsDirectional padding) {
+    final bool singleButton =
+        positiveAction != null && negativeAction == null ||
+            negativeAction != null && positiveAction == null;
+
     return Padding(
       padding: EdgeInsetsDirectional.only(
         top: padding.bottom,
-        start: actionsAxis == Axis.horizontal ? SMALL_SPACE : 0.0,
+        start: (actionsAxis == Axis.horizontal || singleButton)
+            ? SMALL_SPACE
+            : 0.0,
```

Comment:
I have a headache just reading that code.
Perhaps it would make sense to always put SMALL_SPACE to start and end, wouldn't it?